.openpublishing.redirection.json

```diff
@@ -956,6 +956,10 @@
       "source_path": "docs/csharp/methods-lambda-expressions.md",
       "redirect_url": "/dotnet/csharp/programming-guide/statements-expressions-operators/lambda-expressions"
     },
+    {
+      "source_path": "docs/csharp/misc/",
```

Comment:
I don't think this works currently.  @Sampy can you verify?

Reply:
I've sent a couple emails to verify. Folks are still out on holiday. It will be next week before I get clear answers.

Reply:
Thanks Bill! If not, I can hack something together to create entries for each file. 😊

Reply:
There's a workaround we can do that would avoid adding the entry for each file but is not as transparent. If there's no file rename might be easier.

Reply:
@BillWagner is this PR on hold until folder-level redirects are implemented?

Reply:
Yes @tdykstra That's correct.